packages/rancher-istio/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
 packageVersion: 00
-releaseCandidateVersion: 01
+releaseCandidateVersion: 00
```

Comment:
RC is getting decremented?

Reply:
Oh, this is the cause? https://github.com/rancher/charts/pull/990#issuecomment-777633336

Reply:
Its a new minor version, so package and rc version start back at 00.

Reply:
Doesn't package version get incremented then? Or is that not the case since it's the same release? (CC: @aiyengar2)

Reply:
If 1.8.100-rc00 is released in the release branch, and we need to make a change, we would increment package version >> 1.8.101-rc00
If 1.8.100-rc00 is only in dev branch, and we are making changes, we would increment rc version >> 1.8.100-rc01
If minor version does not exist in dev branch or release branch, we create a pr for 1.8.300-rc00 with no package and version increments

Reply:
Add guidance for this in https://github.com/rancher/charts/tree/dev-v2.5-source#versioning-packages

Reply:
Got it. All good then